docs/resources/volume.md

```diff
@@ -29,13 +29,26 @@ resource "databricks_schema" "things" {
   }
 }
 
+resource "databricks_storage_credential" "external" {
+  name = "creds"
+  aws_iam_role {
+    role_arn = "role"
```

Comment:
please give more specifcic on what to give into `role_arn` parameter. Preferably, follow the example of other resources and include resource example how to create that role

Reply:
Updated.